apps/shared/test-helpers/mine.js

```diff
@@ -1,43 +1,49 @@
-const Web3 = require("web3");
-const PrivateKeyProvider = require("truffle-privatekey-provider");
-const provider = new PrivateKeyProvider(
-    "A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563",
-    "http://localhost:8545"
-  );
-const web3 = new Web3(provider);
+/// uncomment the Below block to use on aragon devchain without truffle
 
-const mineBlock = require("./mineBlock")(web3)
-const getBlock = require("./blockNumber")(web3)
+//const Web3 = require("web3");
+//const PrivateKeyProvider = require("truffle-privatekey-provider");
+//const provider = new PrivateKeyProvider(
+//    "A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563",
+//    "http://localhost:8545"
+//  );
+//const web3 = new Web3(provider);
 
-const mineBlocks = async (blocks) => {
-    for (var i = 0; i < blocks; i++) {
-        try {
-            await mineBlock()
-        }
-        catch(e) {
-            console.error('error: ',e)
+const mineBlock = require("./mineBlock")
+const getBlock = require("./blockNumber")
+
+module.exports = async (cb) => {
+    const mineBlockFn = mineBlock(web3)
+    const getBlockFn = getBlock(web3)
+
+    const mineBlocks = async (blocks) => {
+        for (var i = 0; i < blocks; i++) {
+            try {
+                await mineBlockFn()
+            }
+            catch(e) {
+                console.error('error: ',e)
+            }
         }
+        console.log('\nblock',await getBlockFn(), 'mined')
     }
-    console.log('\nblock',await getBlock(), 'mined')
-    process.exit()
-}
 
-const mineToBlock = async (blockNumber) => {
-    while (blockNumber > await getBlock()) {
-        try {
-            await mineBlock()
-        }
-        catch(e) {
-            console.error('error: ',e)
+    const mineToBlock = async (blockNumber) => {
+        while (blockNumber > await getBlockFn()) {
+            try {
+                await mineBlockFn()
+            }
+            catch(e) {
+                console.error('error: ',e)
+            }
         }
+        console.log('\nblock',await getBlockFn(), 'mined')
     }
-    console.log('\nblock',await getBlock(), 'mined')
-    process.exit()
-}
 
-if (!Number(process.argv[2])) {
-    mineToBlock(process.argv[3])
-}
-else {
-    mineBlocks(process.argv[2])
+    if (!Number(process.argv[4])) {
```

Comment:
`process.argv[4]` might be `''`, empty string? Is that correct?

Reply:
it might otherwise be a string, in which case `process.argv[5]` contains the block number we want to mine to

Reply:
Do you think it's best to coerce to `Number`, or would coercing to `Boolean` be a better way to explain what's happening?

Reply:
coercing to a bool wouldn't work because if this is not a number, it'll still be a string of some length, since we'd be expecting  the `to` string in lieu of a non-numerical input